frontend/tests/emailWorkflow.test.ts

```diff
@@ -152,3 +155,86 @@
   // Should be able to open sena-3 project from user dashboard as we are now a member
   await userDashboardPage.openProject('Sena 3', 'sena-3');
 });
+
+test('ask to join project via craete-project page', async ({ page, tempUserInTestOrg }) => {
+    test.setTimeout(TEST_TIMEOUT_2X);
+
+    const { name, email, password } = tempUserInTestOrg;
+
+    await loginAs(page.request, email, password);
+    let dashboardPage = await new UserDashboardPage(page).goto();
+  
+    // Must verify email before being allowed to request project creation
+    await dashboardPage.emailVerificationAlert.assertPleaseVerify();
+    let emailPage = await tempUserInTestOrg.mailbox.openEmail(page, EmailSubjects.VerifyEmail);
+    let pagePromise = emailPage.page.context().waitForEvent('page');
+    await emailPage.clickVerifyEmail();
+    let newPage = await pagePromise;
+    dashboardPage = await new UserDashboardPage(newPage).goto();
+
+    // Create project with similar name to Sena-3
+    const newProjectPage = await dashboardPage.clickCreateProject();
+    await newProjectPage.fillForm({name: 'Sena', code: 'xyz', purpose: 'Testing', organization: 'Test Org'});
+    await expect(newProjectPage.extraProjectsDiv).toBeVisible();
+    await expect(newProjectPage.askToJoinBtn).toBeDisabled();
+    await newProjectPage.extraProjectsDiv.locator('#extra-projects-sena-3').check();
+    await expect(newProjectPage.askToJoinBtn).toBeEnabled();
+    await newProjectPage.askToJoinBtn.click();
+    await expect(newProjectPage.toast('has been sent to the project manager(s)')).toBeVisible();
+
+    // Log in as manager, approve join request.
+    await loginAs(page.request, 'manager');
+    const managerMailbox = new MaildevMailbox('manager@test.com', page.request);
```

Comment:
this isn't going to work in our staging tests unfortunately, this might mean that we need to make another test user which is a manager in order to receive the email.

Reply:
Attempted fix in commit a8d9a37c, but it's not working yet and I've run out of time today. Will continue tomorrow using a8d9a37c as a starting point.